### ISSUE TITLE
When Using CLI, prints [OBJECT] for some Values

### ISSUE THREAD

**Issue by alexanderververis** [zrrrzzt/gpagespeed#2]
Hi,

The CLI tool would be more useful if it printed out everything as JSON. This is what I get now, notice the [Object] references:

```javascript
{ kind: 'pagespeedonline#result',
  id: 'REDACTED',
  responseCode: 200,
  title: 'REDACTED',
  ruleGroups: { SPEED: { score: 69 } },
  pageStats: 
   { numberResources: 98,
     numberHosts: 7,
     totalRequestBytes: '11296',
     numberStaticResources: 48,
     htmlResponseBytes: '150862',
     cssResponseBytes: '393851',
     imageResponseBytes: '737177',
     javascriptResponseBytes: '2286631',
     numberJsResources: 41,
     numberCssResources: 20 },
  formattedResults: 
   { locale: 'en_US',
     ruleResults: 
      { AvoidLandingPageRedirects: [Object],
        EnableGzipCompression: [Object],
        LeverageBrowserCaching: [Object],
        MainResourceServerResponseTime: [Object],
        MinifyCss: [Object],
        MinifyHTML: [Object],
        MinifyJavaScript: [Object],
        MinimizeRenderBlockingResources: [Object],
        OptimizeImages: [Object],
        PrioritizeVisibleContent: [Object] } },
  version: { major: 1, minor: 15 } }
```

**Comment by zrrrzzt** [comment]
Hi @alexanderververis 

I've just released V3.0.0.0 to solve your problem. I recommend to use [json](https://www.npmjs.com/package/json) to prettyprint the result.

```sh
$ gpagespeed <url> --key=<key> | json
```

**Comment by alexanderververis** [comment]
Thanks for this! Works great!
